bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/ContentFragmentUtils.java

```diff
@@ -245,19 +252,32 @@ public static String getGridResourceType(Resource fragmentResource) {
      * @param resourceIterator Iterator of resources for which to get the component exporters.
      * @param modelFactory Model factory service.
      * @param slingHttpServletRequest Current request.
+     * @param callerResource The caller (XF) resource.
```

Comment:
I would be more specific. e.g.:
The caller resource, i.e. the page or template resource referencing the experience fragment or the content fragment.

Reply:
Updated in 4d14728.